jwql/website/apps/jwql/views.py

```diff
@@ -825,13 +829,14 @@ def explore_image(request, inst, file_root, filetype, rewrite=False):
                'suffix': image_info_list['suffixes'][fits_index],
                'num_ints': image_info_list['num_ints'],
                'available_ints': image_info_list['available_ints'],
+               'extensions': extensions,
                'base_url': get_base_url(),
                'form': form}
 
     return render(request, template, context)
 
 
-def explore_image_ajax(request, inst, file_root, filetype, scaling="log", low_lim=None, high_lim=None, rewrite=False):
+def explore_image_ajax(request, inst, file_root, filetype, scaling="log", low_lim=None, high_lim=None, ext_name="SCI", rewrite=False):
```

Comment:
Do we always expect there to be a science extension available?

Reply:
Spoke with @bhilbert4 who confirmed before adding this code.